### ISSUE TITLE
Elixir build failing when used as devcontainer feature

### ISSUE THREAD

**Issue by ccavolt** [version-fox/vfox-elixir#3]
Hi, I'm building devcontainer features for vfox, vfox-erlang, and vfox-elixir. I'm pretty sure I have the first two figured out but I'm stuck on vfox-elixir. When I try to run a build I get the following error:

![image](https://github.com/version-fox/vfox-elixir/assets/10134274/a7d415a4-0e23-4cf1-b4b1-46f04cb51485)

I normally like trying to figure out things myself but I'm not sure what to do here. The error messages are vague and I'm not well versed in Lua or makefiles. If you can provide any help, I'd greatly appreciate it! If there's any additional information that would help, just let me know.

**Comment by yeshan333** [comment]
> Hi, I'm building devcontainer features for vfox, vfox-erlang, and vfox-elixir. I'm pretty sure I have the first two figured out but I'm stuck on vfox-elixir. When I try to run a build I get the following error:
> 
> ![image](https://private-user-images.githubusercontent.com/10134274/328665232-a7d415a4-0e23-4cf1-b4b1-46f04cb51485.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTUxMzQ1NDksIm5iZiI6MTcxNTEzNDI0OSwicGF0aCI6Ii8xMDEzNDI3NC8zMjg2NjUyMzItYTdkNDE1YTQtMGUyMy00Y2YxLWI0YjEtNDZmMDRjYjUxNDg1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTA4VDAyMTA0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTViYmUyMDQ4ZTY5YWNhZDdlMGQ0YWE4YzZkYjQxOGI2MzkzMmYzOTNlMWRkZmI3NWIzODk1Y2IwMTY2ZWZkYzQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.JgvSHHFq-xvF270a8DUtem8cLAZ0HHbcTpuM0pic4gk)
> 
> I normally like trying to figure out things myself but I'm not sure what to do here. The error messages are vague and I'm not well versed in Lua or makefiles. If you can provide any help, I'd greatly appreciate it! If there's any additional information that would help, just let me know.

It seem that the erlang compiler not in $PATH. If you use vfox-erlang to install Erlang/OTP, you must use the `vfox use erlang@xxxx` before you install Elixir.
![image](https://github.com/version-fox/vfox-elixir/assets/39296814/5e6cb88b-1f5b-4863-ae5c-763897034b2f)

Hear are the example installation flow [https://github.com/version-fox/vfox-elixir/blob/5e6fcc5989fc55348cca45fae1bf3bae18201b72/.github/workflows/e2e_test.yaml#L54](https://github.com/version-fox/vfox-elixir/blob/5e6fcc5989fc55348cca45fae1bf3bae18201b72/.github/workflows/e2e_test.yaml#L54):

![image](https://github.com/version-fox/vfox-elixir/assets/39296814/da6fa7f1-b849-4696-b0f7-caa7f8780f5a)

```shell
# like this
vfox use erlang@xxx
vfox install elixir@xxx
```
@ccavolt

**Comment by ccavolt** [comment]
@yeshan333 Thanks for the quick response! In my script, I had included the vfox use command. What ended up being the problem was the "vfox activate bash" command, being in .bashrc, doesn't activate in scripts, so I had to trigger it manually after installing Erlang, and then again after installing Elixir in order to get the PATH to update. I see now that your example installation flow is also calling vfox activate bash after installing each language.

Thanks for pointing me in the right direction and for writing this plugin!
